Content.Shared/_CorvaxNext/Resomi/Abilities/Hearing/ListenUpComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Robust.Shared.Utility;
+
+namespace Content.Shared._CorvaxNext.Resomi.Abilities.Hearing;
+
+[RegisterComponent]
+public sealed partial class ListenUpComponent : Component
+{
+    [DataField]
+    public float radius = 8f;
+
+    [DataField]
+    public SpriteSpecifier.Rsi Sprite = new SpriteSpecifier.Rsi(new("/Textures/_CorvaxNext/Mobs/Species/Resomi/Abilities/noise_effect.rsi"), "noise");
```

Comment:
```suggestion
    public SpriteSpecifier Sprite = new SpriteSpecifier.Rsi(new("/Textures/_CorvaxNext/Mobs/Species/Resomi/Abilities/noise_effect.rsi"), "noise");
```
Для большей совместимости, если я правильно понимаю.